### PR TITLE
[v3] add a sticky banner for helm 2 and helm info

### PR DESCRIPTION
**Pull request by flynnduism**
Adds a sticky banner to the site which states:

* highlights the coming of Helm 3
* states what version of Helm you are currently viewing info on
* links to the other version

### CODE REVIEW

**Comment by flynnduism** [comment]
"You are viewing Helm 2 (latest stable).   Helm 3 is coming soon (alpha-1 is here).   Visit the v3 docs or read the blog for details."

Screenshot:

Screen Shot 2019-05-14 at 2 30 17 PM

---

V3 links go to [v3.helm.sh](https://v3.helm.sh/), which has a different message and in turn links back to V2 (aka http://helm.sh)

File: themes/helm/layouts/partials/banner.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<div id="banner">
+  <div class="cc-container">
+
+    <!-- viewing helm 2 version of site (desktop) -->
+    <p class="center text-center show-for-medium-up">You are viewing Helm 2 (latest stable). &nbsp; Helm 3 is coming soon (alpha-1 is <a href="https://v3.helm.sh"><strong>here</strong>). &nbsp; Visit the <a href="https://v3.helm.sh"><strong>v3 docs</strong></a> or read the <a href="https://helm.sh/blog"><strong>blog</strong></a> for details.</p>
```
Comment (bacongobbler) [inline themes/helm/layouts/partials/banner.html]: What do you think about changing the verbiage to "3.0.0-alpha.1 is here"? Is that too long for a banner?

Comment (flynnduism) [reply themes/helm/layouts/partials/banner.html]: Yeah that makes sense, and is fine in length. Would you rather it as: 

```
You are viewing Helm 2 (latest stable).   Helm 3 is coming soon (3.0.0-alpha.1 is here) [...]
```

or 

```
You are viewing Helm 2 (latest stable).   Helm 3.0.0-alpha.1 is here [...]
```

Comment (bacongobbler) [reply themes/helm/layouts/partials/banner.html]: I like the latter

**Pull request merged**